pkg/code/utils.go

```diff
@@ -37,6 +37,27 @@ func CreateCodeIntegration(ctx context.Context, codeIntegrations []CodeIntegrati
 	return AddCodeIntegration(ctx, name)
 }
 
+func GetCodeIntegrationById(ctx context.Context, codeIntegrationId string) (*CodeIntegration, error) {
+	codeIntegrations, err := GetCodeIntegrations(ctx)
+	if err != nil {
+		return nil, err
+	}
+	selected, err := entity.GetEntityById(codeIntegrationId, codeIntegrations, CodeIntegrationEntityDesc)
+	if err != nil {
+		return nil, fmt.Errorf("failed to get code integration by id %s: %v", codeIntegrationId, err)
+	}
+	return selected, nil
+}
+
+func CodeIntegrationInfo(codeIntegration *CodeIntegration) {
```

Comment:
Consider to rename the function to `PrintCodeIntegrationInfo `

Reply:
done